app/views/coronavirus_landing_page/components/landing_page/_statistics_section.html.erb

```diff
@@ -13,6 +13,8 @@
         <p><%= topic_section.dig("new_positive_tests", "title") %></p>
         <p>Daily</p>
         <p><%= number_with_delimiter(statistics.new_positive_tests) %></p>
+        <p>Last 7 days</p>
```

Comment:
This means that the heading will show up even if there's data missing and `current_week_positive_tests` doesn't exist right? Is that expected behaviour or should we hide the heading in an `if` statement? (extrapolate comment to other cases where it applies)

Reply:
You're right. I think I'm missing a guard clause before the merge, so this:

```ruby
daily_tests = daily_tests(latest_tests)
weekly_tests = weekly_tests(latest_tests, data)

daily_tests.merge!(weekly_tests)
```

Should be:

```ruby
daily_tests = daily_tests(latest_tests)
weekly_tests = weekly_tests(latest_tests, data)

return unless daily_tests && weekly_tests

daily_tests.merge!(weekly_tests)
```

Reply:
I think the same guard is missing for hospital admissions as well.

Reply:
Sounds good :)

Reply:
Thank you for being so thorough with this review. It's much appreciated.